Foundation/include/Poco/NumberFormatter.h

```diff
@@ -43,6 +43,15 @@ class Foundation_API NumberFormatter
 		FMT_ON_OFF
 	};
 
+	enum class Options
+	{
+		OPT_NONE = 0,
+		OPT_PREFIX = 1 << 0,
+			/// Prepend prefix 0x
+		OPT_LOWERCASE = 1 << 1
+			/// Use uppercase letters for hexadecimal values
```

Comment:
OPT_LOWERCASE and comment says "Use UPPERCASE" ???

Reply:
Resolved in new commit.